t/plugins.t

```diff
@@ -16,6 +16,15 @@ for my $file (<./plugins/*.json>) {
         ok !$@, "valid JSON";
         ok $json->{source}, "has source";
         ok $json->{description}, "has description";
+
+        my $valid = ok $json->{source} =~ m|\A[-_.a-zA-Z0-9]+/[-_.a-zA-Z0-9]+\Z|, 'source is valid';
```

Comment:
Should use https://metacpan.org/pod/Test::More#like, it makes more helpful error message.

Reply:
Thank you! fixed at https://github.com/mackerelio/plugin-registry/pull/10/commits/981a6efd4aaced03ae5fa2976915a497b236fecc